proposals/gc/MVP.md

```diff
@@ -285,16 +277,20 @@ Perhaps also the following short-hands:
 
 #### Runtime Types
 
-* `rtt.get <typeuse>` returns the RTT of the specified type
-  - `rtt.get t : [] -> [(rtt t)]`
+* `rtt.none` returns the RTT that cannot be used to cast to anything
+  - `rtt.none : [] -> [(rtt anyref)]`
   - multiple invocations of this instruction yield the same observable RTTs
+
+* `rtt.new <typeuse>` returns the RTT of the specified type
+  - `rtt.new t : [] -> [(rtt t)]`
+  - each invocation of this instruction yields a distinct RTT
+  - equivalent to `(rtt.sub t (rtt.none))`
   - this is a *constant instruction*
-  - equivalent to `(rtt.sub t (rtt.get anyref))`, except when `t` itself is `anyref`
 
 * `rtt.sub <typeuse>` returns the RTT of the specified type as a sub-RTT of a given parent RTT operand
   - `rtt.sub t : [(rtt t')] -> [(rtt t)]`
     - iff `t <: t'`
-  - multiple invocations of this instruction with the same operand yield the same observable RTTs
+  - each invocation of this instruction yields a distinct RTT
```

Comment:
@tlively: Here is the most important change (along with the matching change in `rtt.get` to `rtt.new`). An "applicative" system returns the same value when given the same inputs (previous semantics), whereas a "generative" system returns a new value distinct from all previous values (new semantics).